src/node/clean-up-temp-files.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+'use strict';
+
+const path = require('path');
+const del = require('del');
+const pTry = require('p-try');
+
+const cleanUpTempFiles = (
+  batfishConfig: BatfishConfiguration
+): Promise<void> => {
```

Comment:
`lint-flow` does not like this

Reply:
Say more?

Reply:
I changed this to `Promise<all>` and Flow is happy, but I am not really sure why it works.